### PR TITLE
bugfix-RND-256-new-endpoint-to-clean-disconnected-producers-consumers

### PR DESCRIPTION
**Pull request by shohamroditimemphis**

### CODE REVIEW

File: models/producers.go [hunk 1]
```diff
@@ -35,12 +35,24 @@ type ExtendedProducer struct {
 	Name         string    `json:"name"`
 	Type         string    `json:"type,omitempty"`
 	ConnectionId string    `json:"connection_id,omitempty"`
-	UpdatedAt    time.Time `json:"created_at"`
+	UpdatedAt    time.Time `json:"updated_at"`
```
Comment (idanasulinmemphis) [inline models/producers.go:38]: have you checked that the UI is not expecting this name?

File: server/memphis_handlers_monitoring.go [hunk 2]
```diff
@@ -705,90 +710,93 @@ func (mh MonitoringHandler) GetStationOverviewData(c *gin.Context) {
...
+				"schema":                          emptyResponse,
+				"idempotency_window_in_ms":        station.IdempotencyWindow,
+				"dls_configuration_poison":        station.DlsConfigurationPoison,
+				"dls_configuration_schemaverse":   station.DlsConfigurationSchemaverse,
+				"total_dls_messages":              totalDlsAmount,
+				"tiered_storage_enabled":          station.TieredStorageEnabled,
+				"created_by_username":             station.CreatedByUsername,
+				"resend_disabled":                 station.ResendDisabled,
+				"functions_enabled":               functionsEnabled,
+				"max_amount_of_allowed_producers": usageLimit,
```
Comment (idanasulinmemphis) [inline server/memphis_handlers_monitoring.go:770]: you should add it in the websocket as well

File: db/db.go [hunk 3]
```diff
@@ -2670,44 +2670,46 @@ func GetNotDeletedProducersByStationID(stationId int) ([]models.Producer, error)
...
+    p.connection_id,
+    p.updated_at,
+    s.name,
+    p.is_active,
+	COUNT(CASE WHEN p.is_active THEN 1 END) OVER (PARTITION BY p.name) AS connected_producers_count,
+	COUNT(CASE WHEN NOT p.is_active THEN 1 END) OVER (PARTITION BY p.name) AS disconnected_producers_count
+FROM producers AS p
+LEFT JOIN stations AS s ON s.id = p.station_id
+WHERE p.station_id = $1
+ORDER BY p.name, p.is_active, p.updated_at DESC
```
Comment (idanasulinmemphis) [inline db/db.go:2694]: is it in purpose that you deleted the p.is_active DESC?

**Review by idanasulinmemphis — CHANGES_REQUESTED** [review]

File: db/db.go (hunk 3 above)
Comment (shohamroditimemphis) [reply db/db.go:2694]: no, my mistake. i fixed it.

**Review by idanasulinmemphis — APPROVED** [review]

**Pull request merged**